src/main/java/tc/oc/pgm/PGMImpl.java

```diff
@@ -143,12 +160,13 @@
 import tc.oc.pgm.tracker.TrackerMatchModule;
 import tc.oc.pgm.wool.WoolModule;
 import tc.oc.pgm.worldborder.WorldBorderModule;
+import tc.oc.tablist.TabManager;
 import tc.oc.util.SemanticVersion;
 
 public final class PGMImpl extends JavaPlugin implements PGM {
 
   private MatchManager matchManager;
-  private MatchTabManager matchTabManager;
+  private TabManager tabManager;
```

Comment:
I don't get why this was changed (and the subsequent changes) if you aren't even changing the MatchTabManager for 1.7/1.8+ players.

Reply:
Sorry, I had meant to do something different than what I ended up doing and forgot to undo this.